tests/test_e2e_70_kytos_stats.py

```diff
@@ -82,78 +82,104 @@ def test_010_table_stats(self):
         response = requests.get(api_url)
         assert response.status_code == 200, response.text
         data = response.json()
-        assert len(data) == len(topo_switches)
+        assert len(data) == len(topo_switches), str(data)
         for sw in topo_switches:
-            assert len(data[sw]) == 1
-            assert '0' in data[sw]
+            assert len(data[sw]) == 1, str(data)
+            assert '0' in data[sw], str(data)
 
         api_url = KYTOS_STATS + '/table/stats?table=0&table=1'  
         response = requests.get(api_url)
         assert response.status_code == 200, response.text
         data = response.json()
-        assert len(data) == len(topo_switches)
+        assert len(data) == len(topo_switches), str(data)
         for sw in topo_switches:
-            assert len(data[sw]) == 2
-            assert '0' in data[sw]
-            assert '1' in data[sw]
+            assert len(data[sw]) == 2, str(data)
+            assert '0' in data[sw], str(data)
+            assert '1' in data[sw], str(data)
 
         api_url = KYTOS_STATS + '/table/stats?dpid=00:00:00:00:00:00:00:01&table=0'
         response = requests.get(api_url)
         assert response.status_code == 200, response.text
         data = response.json()
-        assert len(data) == 1
-        assert len(data['00:00:00:00:00:00:00:01']) == 1
-        assert '0' in data['00:00:00:00:00:00:00:01']
+        assert len(data) == 1, str(data)
+        assert len(data['00:00:00:00:00:00:00:01']) == 1, str(data)
+        assert '0' in data['00:00:00:00:00:00:00:01'], str(data)
 
         api_url = KYTOS_STATS + '/table/stats'  
         response = requests.get(api_url)
         assert response.status_code == 200, response.text
         data = response.json()
-        assert len(data) == len(topo_switches)
+        assert len(data) == len(topo_switches), str(data)
 
     def test_015_packet_count(self):
         """Test packet_count""" 
+        sw = "00:00:00:00:00:00:00:01"
 
-        api_url = KYTOS_STATS + '/flow/stats'  
+        # install a flow
+        cookie = 5
+        payload = {"flows": [{"cookie": cookie, "packet_count": 8, "duration_sec": 2}]}
```

Comment:
packet_count and duration_sec shouldn't be sent, it doesn't cause any issues (now), but can get confusing for anyone else reading this in the future, can you review this?